### DISCUSSION TITLE
Indexing Database reset

### DISCUSSION THREAD

**Discussion by tkrommydas** [eirikpre/VSCode-SystemVerilog#153] [Q&A] [answered]
Hello.
I have the plugin installed and would like to reset the database in order to reindex all files, because due to wrong exclusionList it indexed about 800.000 objects
I did not find anywhere the database in order to remove it.
Could you please help me with that?
I am using the plugin on remote host running Linux.
Thank you very much

**Comment by joecrop** [comment, marked as answer]
To reindex, all you need to do is click the SystemVerilog button at the bottom of VSCode: 
![image](https://user-images.githubusercontent.com/3010369/161802597-8251441c-7394-4fb8-9491-92d858b7ecbd.png)

**Reply by tkrommydas** [reply]
Thank you very much for your swift answer!
